packages/@webex/internal-plugin-metrics/src/rtcMetrics/index.ts

```diff
@@ -40,18 +58,23 @@ export default class RtcMetrics {
    * Initialize the interval.
    *
    * @param {object} webex - The main `webex` object.
-   * @param {string} meetingId - The meeting id.
+   * @param {IdType} Ids - Meeting or Calling id.
    * @param {string} correlationId - The correlation id.
    */
-  constructor(webex, meetingId, correlationId) {
+  constructor(webex, {meetingId, callId}: IdType, correlationId) {
     // `window` is used to prevent typescript from returning a NodeJS.Timer.
     this.intervalId = window.setInterval(this.sendMetricsInQueue.bind(this), 30 * 1000);
     this.meetingId = meetingId;
+    this.callId = callId;
     this.webex = webex;
     this.correlationId = correlationId;
     this.resetConnection();
   }
 
+  public updateCallId(callId: string) {
```

Comment:
Did we need this to update callId for outbound call cases since callId is not generated initially?

Reply:
@Kesari3008 yes, please see my comment above

Reply:
Done